zenith_utils/src/postgres_backend.rs

```diff
@@ -368,6 +368,10 @@ impl PostgresBackend {
                             AuthType::Trust => {
                                 self.write_message_noflush(&BeMessage::AuthenticationOk)?
                                     .write_message_noflush(&BeParameterStatusMessage::encoding())?
+                                    // The async python driver requires a valid server_version
```

Comment:
This is pretty ad-hoc. I know, I wrote it myself, but for something to actually commit, let's at least improve the comment here. Would the async python driver accept something like `14pageserver`, to make it more clear that it's not an actual Postgres server?

Reply:
Looks like this comment is related to some other PR, isn't it?